docs/snippets/react/page-story-args-within-story.ts.mdx

```diff
@@ -0,0 +1,36 @@
+```ts
```

Comment:
@jonthenerd sorry for not mentioning it earlier, but could you also provide an example in the form of a snippet aimed at TypeScript 4.9? You don't need to link it, as it will be automatically picked up when the documentation is published; you can name it as `page-story-args-within-story.ts-4-9.mdx` and place it in the same folder as this example (e.g., `react`).

Reply:
done!